mlir/test/Dialect/MIOpen/lowering_input_tensor_non_zero_padding.mlir

```diff
@@ -25,6 +25,7 @@ func @miopen_conv2d_gcyxk_gcnhw_gknhw(%filter : memref<1x8x3x3x128xf32>, %input
 
 // AFFINE: #map{{[0-9]+}} = affine_map<(d0, d1, d2) -> (d0, d1 floordiv 9, (d1 mod 9) floordiv 3, (d1 mod 9) mod 3, d2)>
 // AFFINE-NEXT: #map{{[0-9]+}} = affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d2, d3 - 1, d4 - 1)>
+// AFFINE-NEXT: #map{{[0-9]+}} = affine_map<(d0, d1, d2, d3, d4) -> (0, 0, 0, 1, 1)>
```

Comment:
right now we only have cases with padding on all 4 directions.

more tests in this file is needed to check other cases.

Reply:
done

Reply:
@ltqin ping? I don't see this being done.

I imagine we should have cases like:

```
  miopen.conv2d(%filter, %input, %output) {
    arch = "gfx906",
    num_cu = 64,
    filter_layout = ["g", "c", "y", "x", "k"],
    input_layout = ["gi", "ci", "ni", "hi", "wi"],
    output_layout = ["go", "ko", "no", "ho", "wo"],
    dilations = [1, 1],
    strides = [1, 1],
    padding = [1, 1, 1, 1]
  } : memref<1x8x3x3x128xf32>, memref<1x8x128x32x32xf32>, memref<1x128x128x32x32xf32>
  ```
  where `padding` attribute goes from `[0, 0, 0, 0]` to `[1, 1, 1, 1]` covering 16 different cases.
  
  and the affine map for h/w dimension shall be changed accordingly.